### DISCUSSION TITLE
Environment Secrets - Please add a reveal/hide toggle widget

### DISCUSSION THREAD

**Discussion by ppmgitrepo** [PrestaShop/PrestaShop#35806] [Ideas]
### The problem

When entering an "Environment Secret" variable against an Environment in Git the secret value is always visible during data entry. 
This is an issue because many times the secret values are set by teams that own the password and they do it over a screen share. 

Since the entered value is not masked we cannot do a screen share method, instead need to give Admin access to the Env to that member which is not desirable. 

So request you to please include a password hide/reveal toggle widget so that they can hide it during data entry on a screen share.  

### The solution

Request you to please include a password hide/reveal toggle widget so that they can hide it during data entry on a screen share.  

### Alternatives

The alternative is that the user who has to enter the Secret value should be given Admin access on that repository that owns the Environment. 

### Do you plan to work on this subject?

- [ ] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [ ] I'm willing to provide any wireframes or design assets required for this feature.
- [ ] I'm willing to submit a Pull Request that implements this feature.
- [ ] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by matks** [comment]
Hello @ppmgitrepo are you sure you created that item on the right repository? This is the PrestaShop project and we do not control the GitHub Actions workflow

I think you should rather write your discussion here https://github.com/orgs/community/discussions

**Reply by ppmgitrepo** [reply]
Sorry you may close this discussion
